### ISSUE TITLE
How can i install missing UI modules like Tab

### ISSUE THREAD

**Issue by pavewaytech** [nativescript-vue/nativescript-vue#528]
Version

2.2.2
Reproduction link

Just install the default nativescript-vue template
Platform and OS info

Android / nativescript 5.4.2
Steps to reproduce

Install nativescript-vue version.

Run npm run android --hmr.

Then you get the errors below in the terminal.

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/bottom-navigation' in '/home/erss400/Desktop/projects/zuumpay/ZuumPayMaster/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9315:9-57
 @ ./main.js

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/tab-navigation-base/tab-content-item' in '/home/erss400/Desktop/projects/zuumpay/ZuumPayMaster/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9345:9-76
 @ ./main.js

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/tab-navigation-base/tab-strip' in '/home/erss400/Desktop/projects/zuumpay/ZuumPayMaster/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9333:9-69
 @ ./main.js

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/tab-navigation-base/tab-strip-item' in '/home/erss400/Desktop/projects/zuumpay/ZuumPayMaster/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9339:9-74
 @ ./main.js

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/tabs' in '/home/erss400/Desktop/projects/zuumpay/ZuumPayMaster/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9324:9-44
 @ ./main.js

How can i fix the errors on the terminal because the app builds successfully on the device.

**Comment by tdhman** [comment]
I have encountered the same problem with missing module in nativescript-vue while the application is built and installed successfully on device.
Before I don't have this issue, don't know why....

```
ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/bottom-navigation' in '/Users/htp1/Documents/workspace-ns/ns-vue-3dmessaging/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9315:9-57
 @ ./main.js

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/tab-navigation-base/tab-content-item' in '/Users/htp1/Documents/workspace-ns/ns-vue-3dmessaging/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9345:9-76
 @ ./main.js

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/tab-navigation-base/tab-strip' in '/Users/htp1/Documents/workspace-ns/ns-vue-3dmessaging/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9333:9-69
 @ ./main.js

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/tab-navigation-base/tab-strip-item' in '/Users/htp1/Documents/workspace-ns/ns-vue-3dmessaging/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9339:9-74
 @ ./main.js

ERROR in ../node_modules/nativescript-vue/dist/index.js
Module not found: Error: Can't resolve 'tns-core-modules/ui/tabs' in '/Users/htp1/Documents/workspace-ns/ns-vue-3dmessaging/node_modules/nativescript-vue/dist'
 @ ../node_modules/nativescript-vue/dist/index.js 9324:9-44
 @ ./main.js
```

**Comment by tdhman** [comment]
Maybe this issue is related to the same issue in NS Angular: [https://github.com/NativeScript/nativescript-angular/issues/1914](https://github.com/NativeScript/nativescript-angular/issues/1914)

**Comment by fhackenb** [comment]
Did any of you guys figure this out? @tdhman @pavewaytech

**Comment by czachandrew** [comment]
It seems that even though these errors are thrown tabs and the like seem to function appropriately... same for you? Very strange

**Comment by tdhman** [comment]
@fhackenb I have to fix the version of "nativescript-vue" to "2.2.2" for TNS 5.4.2
Unless you upgrade your TNS 6.0 then the newest version of "nativescript-vue" will be compatible.

**Comment by rigor789** [comment]
`2.3.0` should also work for TNS 5.4. `2.4.0` is the TNS 6.* compatible version, and it includes the new tab UI components.
